source/draw/RasterYuv422Semipacked.ooc

```diff
@@ -82,13 +86,10 @@ RasterYuv422Semipacked: class extends RasterPacked {
 		}
 	}
 	open: static func (filename: String) -> This {
-		x, y, imageComponents: Int
-		requiredComponents := 3
-		data := StbImage load(filename, x&, y&, imageComponents&, requiredComponents)
-		bgr := RasterBgr new(ByteBuffer new(data as UInt8*, x * y * requiredComponents), IntSize2D new(x, y))
-		result := This new(bgr)
+		bgr := RasterBgr open(filename)
```

Comment:
Anyway, StbImage will load a PNG or JPG. JPG can only be BGR afaik, while PNG can also be BGRA and possibly monochrome as well. StbImage will then convert the image from whichever number of channels into 3 channels (since we're opening it as a BGR), which will then be converted by us into YUV. It wouldn't make sense to keep the alpha channel when decoding the image since YUV doesn't have such a channel, and the only downside to assuming the image is BGR and not monochrome is that StbImage will unnecessarily convert it from monochrome to BGR only to let us convert it back to a monochrome YUV.